app/views/stories/new.html.erb

```diff
@@ -20,6 +20,11 @@
       default: 'Conservative',
       input_html: { class: 'rounded mt-5', style: 'margin-left: 22px;' } %>
     <%= f.hidden_field :body, input_placeholder: 'AI will generate this' %>
+    <div>
+      LLM Name <%= select_tag "llm_name", options_for_select(LLMS), 
```

Comment:
@mattlindsey, any suggestions about a better way to implement this?

Reply:
That looks good to me. Since we didn't add llm_name to the stories table we can't do `f.input :llm_name, collection: LLMS`.

Reply:
Actually it would be cleaner to add llm_name to the stories table, so you can do that if you want (and update the story create job). But this should be fine too.